plugins/extension/joomlaupdate/src/Extension/Joomlaupdate.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  Extension.joomla
+ *
+ * @copyright   (C) 2025 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Plugin\Extension\Joomlaupdate\Extension;
+
+use Joomla\CMS\Event\Model\AfterSaveEvent;
+use Joomla\CMS\Plugin\CMSPlugin;
+use Joomla\Component\Joomlaupdate\Administrator\Model\UpdateModel;
+use Joomla\Event\SubscriberInterface;
+
+// phpcs:disable PSR1.Files.SideEffects
+\defined('_JEXEC') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+/**
+ * The extension plugin for com_joomlaupdate
+ *
+ * @since   __DEPLOY_VERSION__
+ */
+final class Joomlaupdate extends CMSPlugin implements SubscriberInterface
+{
+    /**
+     * Load the language file on instantiation.
+     *
+     * @var    boolean
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+    protected $autoloadLanguage = true;
+
+    /**
+     * Returns an array of events this subscriber will listen to.
+     *
+     * @return array
+     *
+     * @since   __DEPLOY_VERSION__
+     */
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            'onExtensionAfterSave' => 'onExtensionAfterSave'
```

Comment:
```suggestion
            'onExtensionAfterSave' => 'onExtensionAfterSave',
```
Fix PHPCS (rule trailing_comma_in_multiline)